menoh/node.hpp

```diff
@@ -20,6 +20,7 @@ namespace menoh_impl {
     };
 
     bool operator==(node const& a, node const& b);
+    inline bool operator!=(node const& a, node const& b) { return !(a == b); }
```

Comment:
Is this necessary for the objective of this PR?
If not, how about removing this from this PR and applying the change separately?

Reply:
You are correct. It is not necessary for this PR. I will remove it.